src/qml/bitcoin.cpp

```diff
@@ -67,6 +67,7 @@ bool InitErrorMessageBox(
 {
     QQmlApplicationEngine engine;
     engine.rootContext()->setContextProperty("message", QString::fromStdString(message.translated));
+    engine.addImageProvider(QStringLiteral("images"), new ImageProvider{NULL});
```

Comment:
Did you mean to add this change to the commit?

Reply:
thanks for spotting that, this was from an earlier iteration of the PR where we were creating a custom os independent title bar for the window; we're no longer doing that. removed